package.json

```diff
@@ -46,7 +46,8 @@
     "test": "npm run lint && npm run test-unit && npm run test-integration-runner",
     "lint": "eslint .",
     "autofix": "eslint --fix .",
-    "codecov": "npm run test && (codecov || true)"
+    "codecov": "npm run test && (codecov || true)",
+    "validate": "npm ls"
```

Comment:
What's the reason behind adding `validate`

Reply:
I think npm-check-updates added that automatically. I can remove it if we don't usually include it.